### PR TITLE
[fullscreen] feat: changing bg and icon colors

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: packages/fullScreen/web/src/icons/expand.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable max-len */
+import React from 'react';
+
+const ExpandIcon = props => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="60"
+    height="60"
+    viewBox="0 0 60 60"
+    // style="background:#fff"
```
Comment (talevy17) [inline packages/fullScreen/web/src/icons/expand.tsx:10]: remove

File: packages/fullScreen/web/src/icons/expand.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable max-len */
+import React from 'react';
```
Comment (shaulgo) [inline packages/fullScreen/web/src/icons/expand.tsx:2]: all icons - the convention for react component files is to start with capital letter.

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: packages/fullScreen/web/src/fullscreen.rtlignore.scss [hunk 3]
```diff
@@ -34,6 +34,22 @@
...
+  }
+}
+
+.nav_arrow_right {
+  left: 22px;
+  position: absolute;
+  > svg {
+    filter: drop-shadow(0px 0px 0px black) !important;
+  }
+}
```
[the remark this answers is not in the document]
Comment (talevy17) [reply packages/fullScreen/web/src/fullscreen.rtlignore.scss:51]: yes, gallery adds a drop shadow that Nitzan didn't like

File: docs/docs/ricos/ricos-api.mdx [hunk 4]
```diff
@@ -86,10 +86,19 @@ Any two letter string represing a [supported](https://github.com/wix/ricos/tree/
...
+default colors are: <br />
+backgroundColor: 'white';
+iconsColor: 'black'; <br />
+
 ```jsx
 mediaSettings?: {
   pauseMedia?: boolean;
   disableRightClick?: boolean;
+  fullscreenProps?: {
+  backgroundColor?: string;
```
[the remark this answers is not in the document]
Comment (talevy17) [reply docs/docs/ricos/ricos-api.mdx:99]: yes it's a color string, I'll add to the docs

File: packages/fullScreen/web/src/InnerFullScreen.jsx [hunk 5]
```diff
@@ -195,7 +216,7 @@ InnerFullscreen.propTypes = {
   isMobile: PropTypes.bool,
   index: PropTypes.number,
   topMargin: PropTypes.object,
-  backgroundColor: PropTypes.object,
-  foregroundColor: PropTypes.object,
+  backgroundColor: PropTypes.string,
+  iconsColor: PropTypes.string,
```
[the remark this answers is not in the document]
Comment (talevy17) [reply packages/fullScreen/web/src/InnerFullScreen.jsx:220]: yes it does, in expand mode - I'll change iconsColor to foreground and apply for titles as well

**Review by shaulgo — APPROVED** [review]

**Pull request merged**